amundsen_application/static/js/components/NavBar/tests/index.spec.tsx

```diff
@@ -107,27 +108,35 @@ describe('NavBar', () => {
       expect(spy).toHaveBeenCalledWith(AppConfig.navLinks);
     });
 
-    it('renders Avatar for loggedInUser', () => {
-      expect(wrapper.find(Avatar).props()).toMatchObject({
-        name: props.loggedInUser.display_name,
-        size: 32,
-        round: true,
-      })
-    });
+    describe('if indexUsers is enabled', () => {
+      it('renders Avatar for loggedInUser inside of user dropdown', () => {
+        expect(wrapper.find(Dropdown).find(Dropdown.Toggle).find(Avatar).props()).toMatchObject({
+          name: props.loggedInUser.display_name,
+          size: 32,
+          round: true,
+        })
+      });
 
-    it('renders a Link to the user profile if `indexUsers` is enabled', () => {
-      expect(wrapper.find('#nav-bar-avatar-link').exists()).toBe(true)
+      it('renders user dropdown header', () => {
+        element = wrapper.find(Dropdown).find(Dropdown.Menu).children().at(0);
```

Comment:
Could you do something like `element = wrapper.find('.profile-menu-header')`? I think this would be more explicit

Reply:
Will do.